pytest.ini

```diff
@@ -20,6 +20,7 @@ markers =
     monitoring: monitoring related tests
     workloads: workloads related tests
     flowtests: flowbased related tests
+    system_test: systemtest related tests
```

Comment:
Could you explain what a system test in this context means exactly?

Reply:
The test related to system level scenarios where the test scenarios execerise multiple features together.
Added the same in the marker description as well.